### PR TITLE
Feature 11 editor for layout component properties

### PR DESCRIPTION
**Pull request by Jdallos**

### CODE REVIEW

**Review by EVP1989 — COMMENTED** [review]
Added comments

File: src/styles/components/generic/element-settings-pane.ts [hunk 1]
```diff
@@ -49,7 +49,7 @@ export default createStyles({
     }
   },
   content: {
-    padding: theme.spacing(2),
+    // padding: theme.spacing(2),
```
[the remark this answers is not in the document]
Comment (Jdallos) [reply src/styles/components/generic/element-settings-pane.ts:52]: removed

File: src/components/layout/editor-components/html/layout-component-properties.tsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+      <Divider sx={{ color: "#F5F5F5" }}/>
+      <PropertyBox>
+        { renderPanelSubtitle(strings.layout.htmlProperties.layoutProperties.contentEmphasis) }
+        <AlignmentEditorHtml
+          onChange={ onPropertyChange }
+        />
+      </PropertyBox>
+		</Stack>
+	);
+};
```
[the remark this answers is not in the document]
Comment (Jdallos) [reply src/components/layout/editor-components/html/layout-component-properties.tsx:46]: Very strange, the indentation shown in github is different to what I'm seeing locally and there are no changes to commit....

File: src/components/layout/editor-components/html/generic/render-panel-subtitle.tsx [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+	<Typography
+		fontWeight={500}
+		fontSize="14px"
+		marginBottom={theme.spacing(1)}
+		color="#000000"
+		sx={{ opacity: 0.6 }}
+	>
+		{subtitle}
+	</Typography>
+);
```
[the remark this answers is not in the document]
Comment (Jdallos) [reply src/components/layout/editor-components/html/generic/render-panel-subtitle.tsx:19]: Very strange, the indentation shown in github is different to what I'm seeing locally and there are no changes to commit....

File: src/components/layout/editor-components/html/layout-component-properties.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (Jdallos) [reply src/components/layout/editor-components/html/layout-component-properties.tsx:46]: Very strange, the indentation shown in github is different to what I'm seeing locally and there are no changes to commit....

File: src/components/layout/editor-components/html/alignment-editor-html.tsx [hunk 4]
```diff
@@ -0,0 +1,181 @@
...
+};
+
+/**
+ * Components props
+ */
+interface Props {
+  onChange: (name: string, value: string) => void;
+}
+
+// TODO: Not yet implemented as a controlled componenet
```
[the remark this answers is not in the document]
Comment (Jdallos) [reply src/components/layout/editor-components/html/alignment-editor-html.tsx:23]: After speaking with Ville we decided to leave this component as uncontrolled for now, I though maybe leaving this comment might be useful?

**Review by EVP1989 — APPROVED** [review]
Approved

**Pull request merged**